### PR TITLE
event content format

### PR DESCRIPTION
**Pull request by ibrahiminfinite**
This PR formats the content/details of the event.
The formatted view can be seen in message_format.png

### CODE REVIEW

File: matrix-vis.js [hunk 1]
```diff
@@ -105,11 +105,24 @@ function initGraph() {
     network = new vis.Network(container, data, options);
     // add event listeners
     network.on('select', function(params) {
-        var text = "";
+        var Type = " ";
+        var Sender = " ";
```
Comment (kegsay) [inline matrix-vis.js]: Variable names in Javascript should always start with a lower-case letter. Class names with an upper-case letter. Constants in ALL_CAPS. Please lower case `Type` and `Sender`.

File: matrix-vis.js [hunk 2]
```diff
@@ -105,11 +105,24 @@ function initGraph() {
...
+        var m = " ";
         if (params.nodes.length === 1) {
-            text = JSON.stringify(nodes.get(params.nodes[0]).blob, undefined, 2);
+            Type=JSON.stringify(nodes.get(params.nodes[0]).blob.type, undefined, 2);
+            Sender=JSON.stringify(nodes.get(params.nodes[0]).blob.sender, undefined, 2);
```
Comment (kegsay) [inline matrix-vis.js]: You do not need to `JSON.stringify` this. The JSON object is called `blob` and you're now grabbing the `string` under the keys `type` and `sender`. They are already strings.

File: matrix-vis.js [hunk 3]
```diff
@@ -105,11 +105,24 @@ function initGraph() {
...
+            t1="Event Type : "+Type;
+            t2="Creator : "+Sender;
+            if (Type==JSON.stringify("m.room.member")) {
+            m=JSON.stringify(nodes.get(params.nodes[0]).blob.content.membership, undefined, 2);
+            t3="Action : "+m;
+            }
```
Comment (kegsay) [inline matrix-vis.js:115]: **Indentation!**

You MUST indent when you write code! It makes it _a lot_ harder to read otherwise.

File: matrix-vis.js [hunk 4]
```diff
@@ -105,11 +105,24 @@ function initGraph() {
...
+            else if (Type==JSON.stringify("m.room.message")) {
+            m=JSON.stringify(nodes.get(params.nodes[0]).blob.content.body, undefined, 2);
+            t3="Message : "+m;
+            }
         }
-        document.getElementById('eventInfo').innerHTML = text;
+        document.getElementById('eventInfo').innerHTML = t1 +"<br>"+ t2 +"<br>"+ t3;
```
Comment (kegsay) [inline matrix-vis.js]: It would be better to store `t1 t2 t3` in an array and then insert the `<br>` programmatically via `array.join("<br>")`. That way you won't need to keep updating this line every time you decide to add a new line to the output.

File: matrix-vis.js [hunk 5]
```diff
@@ -105,11 +105,24 @@ function initGraph() {
     network = new vis.Network(container, data, options);
     // add event listeners
     network.on('select', function(params) {
-        var text = "";
+        var Type = " ";
+        var Sender = " ";
+        var m = " ";
```
Comment (kegsay) [inline matrix-vis.js]: This seems to exist only to be used as a temporary variable. You don't need it. In-line your assignments instead:

```
t3 = "Action : " + nodes.get(params.nodes[0]).blob.content.membership;
```

**Comment by kegsay** [comment]
Please can you add more spaces around your assignment operators. E.g `var foo = "bar";` rather than `var foo="bar";`

File: matrix-vis.js [hunk 6]
```diff
@@ -105,11 +105,24 @@ function initGraph() {
...
         if (params.nodes.length === 1) {
-            text = JSON.stringify(nodes.get(params.nodes[0]).blob, undefined, 2);
+            Type=JSON.stringify(nodes.get(params.nodes[0]).blob.type, undefined, 2);
+            Sender=JSON.stringify(nodes.get(params.nodes[0]).blob.sender, undefined, 2);
+            t1="Event Type : "+Type;
+            t2="Creator : "+Sender;
+            if (Type==JSON.stringify("m.room.member")) {
+            m=JSON.stringify(nodes.get(params.nodes[0]).blob.content.membership, undefined, 2);
```
Comment (kegsay) [inline matrix-vis.js]: You constantly do `nodes.get(params.nodes[0]).blob` - it may be worth you pulling this out to a separate variable so you don't have to keep typing it, e.g. `var event = nodes.get(params.nodes[0]).blob;`

File: bundle.js [hunk 7]
```diff
@@ -44,7 +43,7 @@ function initialSync() {
             var room = data.rooms[i];
             if (room.room_id === roomId) {
                 // add new events to graph
-                console.log("Adding " + room.messages.chunk.length + " new events to the graph");
+                console.log("Adding " + room.messages.chunk.length + "new events to the graph");
```
Comment (Kegsay) [inline bundle.js]: Why did you remove this space? Won't this now log things like:
```
Adding 5new events to the graph
```

File: matrix-vis.js [hunk 8]
```diff
@@ -105,11 +105,20 @@ function initGraph() {
     network = new vis.Network(container, data, options);
     // add event listeners
     network.on('select', function(params) {
-        var text = "";
         if (params.nodes.length === 1) {
-            text = JSON.stringify(nodes.get(params.nodes[0]).blob, undefined, 2);
+            t1="Event Type : "+nodes.get(params.nodes[0]).blob.type;
+            t2="Creator : "+nodes.get(params.nodes[0]).blob.sender;
```
Comment (kegsay) [inline matrix-vis.js]: Please put spaces around your operators. E.g. `t1 = "Event Type: " + nodes` rather than `t1="Event Type: "+nodes`

File: matrix-vis.js [hunk 9]
```diff
@@ -105,11 +105,20 @@ function initGraph() {
...
+            if (nodes.get(params.nodes[0]).blob.type=="m.room.member") {
```
Comment (kegsay) [inline matrix-vis.js]: Spaces around operator `==`

File: matrix-vis.js [hunk 10]
```diff
@@ -105,11 +105,20 @@ function initGraph() {
     network = new vis.Network(container, data, options);
     // add event listeners
     network.on('select', function(params) {
-        var text = "";
         if (params.nodes.length === 1) {
-            text = JSON.stringify(nodes.get(params.nodes[0]).blob, undefined, 2);
+            t1="Event Type : "+nodes.get(params.nodes[0]).blob.type;
```
Comment (kegsay) [inline matrix-vis.js]: `t1 t2 t3` are all being declared as global variables, please put `var` on them to scope them to the function.

**Comment by kegsay** [comment]
Getting better: Please re-build `bundle.js`!

**Comment by kegsay** [comment]
LGTM
